docs/external_access.md

```diff
@@ -0,0 +1,89 @@
+
+### Retrieve the CA and create configmaps and secrets
+
+Use the files that you found in the previous step. Replace ```<your-namespace>``` with your chosen namespace
+
+```sh
+kubectl create configmap ca-config-map --from-file=ca.crt --namespace <your-namespace>
+
+kubectl create secret tls ca-key-pair  --cert=ca.crt  --key=ca.key --namespace <your-namespace>
+```
+
+### Create the Cert Manager issuer and secret
+
+Edit the files in ```config/samples/external_access``` -- ```cert-manager-issuer.yaml``` and ```cert-manager-certificate.yaml``` to replace ```<your-namespace>``` with the namespace you are using for the deployment. In ```cert-manager-certificate.yaml``` replace ```<mongodb-name>``` with your MongoDB deployment name. Also replace ```<domain-rs-1>```, ```<domain-rs-2>```, and ```<domain-rs-3>``` with the external FQDNs of the MongoDB replicaset members. Please remember that you will have to add an equal number of entries for each member of the replicaset.
+
+Apply the manifests.
+
+```sh
+kubectl apply -f config/samples/external_access/cert-manager-issuer.yaml
+kubectl apply -f config/samples/external_access/cert-manager-certificate.yaml
+```
+
+### Create the MongoDB deployment
+
+Edit ```config/samples/external_access/mongodb.com_v1_mongodbcommunity_cr.yaml```. Replace <mongodb-name> with the desired MongoDB deployment name -- this should be the same as in the previous step. Replace ```<domain-rs-1>```, ```<domain-rs-2>```, and ```<domain-rs-3>``` with the external FQDNs of the MongoDB replicaset members. Please remember that you should have the same number of entries in this section as the number of your replicaset members. You can also edit the ports for external access to your preferred numbers in this section -- you will remmeber to change them in the next step too. Change ```<your-admin-password>``` to your desired admin password for MongoDB.
+
+Apply the manifest.
+
+```sh
+kubectl apply -f config/samples/external_access/mongodb.com_v1_mongodbcommunity_cr.yaml
+```
+
+Wait for the replicaset to be available.
+
+### Create the external NodePort services for accessing the MongoDB deployment from outside the Kubernetes cluster
+
+Edit ```config/samples/external_access/external_services.yaml``` and replace ```<mongodb-name>``` with the MongoDB deployment name that you have used in the preceeding steps. You can change the ```nodePort``` and ```port``` to reflect the changes (if any) you have made in the previous steps.
+
+Apply the manifest.
+
+```sh
+kubectl apply -f config/samples/external_access/external_services.yaml
+```
+
+### Retrieve the certificates from a MongoDB replicaset member
+
+```sh
+kubectl exec --namespace mcommunity -it <mongodb-name>-0 -c mongod -- bash
+```
+
+Once inside the container ```cat``` and copy the contents of ```/var/lib/tls/ca/ca.crt``` into a file on your local system. Also copy the contents of the ```.pem``` file in ```/var/lib/tls/server``` into a file on your local system.
```

Comment:
[nit]  We already have a CA creating from a file that is already on disk. The `.pem` file should be the only thing we need to access from the pod.

Reply:
Good point. Resolved in latest commit.